packages/zoltan2/core/src/problems/Zoltan2_PartitioningProblem.hpp

```diff
@@ -50,6 +50,8 @@
 #ifndef _ZOLTAN2_PARTITIONINGPROBLEM_HPP_
 #define _ZOLTAN2_PARTITIONINGPROBLEM_HPP_
 
+#define ZOLTAN2_TASKMAPPING_MOVE
```

Comment:
Need comments for this macro; what's happening here?

Reply:
This was a previous variable that was never defined that I have now removed.